### ISSUE TITLE
Clear your timeline

### ISSUE THREAD

**Issue by beforeadream** [IonicaBizau/reset-your-facebook-account#6]
Script to hide all activities from timeline:

Go here:
https://www.facebook.com/your-facebook-username/allactivity?privacy_source=activity_log&log_filter=all

Paste this:
```JavaScript
setInterval (function () {
	$("html, body").animate({ scrollTop: $(document).height() }, "slow");

	$('*[aria-label="Hidden from Timeline"]').each(function () {
		$(this).parents('.bottomborder').remove();
	});

	var allowed = $('*[aria-label="Allowed on Timeline"] span').last();

	allowed.click();

	$('.uiContextualLayer').find('li:nth-child(2)').find('a span').click();

	allowed.parents('.bottomborder').remove();
}, 400);
```

In the end of a day you will get this:

![screen shot 2016-02-01 at 1 24 43](https://cloud.githubusercontent.com/assets/864218/12705944/d99e75f8-c882-11e5-883a-84faa8ebae22.png)

**Comment by IonicaBizau** [comment]
@beforeadream Can you create a pull request with this? Thanks! :sparkle:

**Comment by IonicaBizau** [comment]
Addressed in #7. :+1:
